src/hooks/useTransactions.tsx

```diff
@@ -93,3 +70,22 @@ export function useTransactions() {
   }
   return context;
 }
+
+export function useLatestTransaction() {
+  const {
+    transactions: [latest],
+  } = useTransactions();
+
+  const state = useTransactionState(latest ? latest.hash : "");
```

Comment:
this is a bit weird to pass empty string, but i guess it would just return undefined if there are no transactions? hopefully not an error.

Reply:
this is weird, you're right, I've changed it to accept an undefined string and deal with it